<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpSetDialog</class>
 <widget class="QDialog" name="TcpSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tcp Set</string>
  </property>
  <widget class="QPushButton" name="btn_startaccept">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_port">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:5000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ldt_port">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stopaccept">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>停止监听</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setport">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
